### PR TITLE
LS_container-registry-get-started-terraform.md

### PR DESCRIPTION
**Pull request by LiSeda**

### CODE REVIEW

**Comment by LiSeda** [comment]
@TomArcherMsft Since the terraform sample and include links seem to be current, are you able to weigh in on why the build report is flagging them? Thank you.

**Comment by TomArcherMsft** [comment]
> @TomArcherMsft Since the terraform sample and include links seem to be current, are you able to weigh in on why the build report is flagging them? Thank you.

1. Open the **azure-docs-pr** **.openpublishing.publish.config.json**
2. From within the **dependent_repositories** section, copy the blocks for **terraform_samples** and **azure-dev-docs-pr**
3. Open the **azure-management-docs-pr** **.openpublishing.publish.config.json**
4. Paste in the blocks within the **dependent_repositories** section

You are seeing this problem as recently a lot of documentation were moved out of azure-docs-pr into new repos. Unless those repos have articles pointing to azure-dev-docs-pr or the terraform sample repo, they won't have these repositories. Therefore, this is something you might have to do for other repos.

File: .openpublishing.publish.config.json [hunk 1]
```diff
@@ -67,6 +67,18 @@
       "url": "https://github.com/MicrosoftDocs/azure-docs-pr",
       "branch": "main",
       "branch_mapping": {}
+    },
+    {
+      "path_to_root": "azure-dev-docs-pr",
```
Comment (tejaswikolli-web) [inline .openpublishing.publish.config.json:72]: azure-management-docs-pr

Comment (LiSeda) [reply .openpublishing.publish.config.json:72]: @tejaswikolli-web, azure-dev-docs-pr points to some of the files mentioned in the article and the dependent repos involved. Pls. confirm if you're asking to add a self-referring entry (azure-management-docs-pr) to the other blocks in the file.

Comment (TomArcherMsft) [reply .openpublishing.publish.config.json:72]: @LiSeda, @tejaswikolli-web is correct. You're in the public repo. You need to fork the private repo: https://github.com/MicrosoftDocs/azure-management-docs-pr

Comment (LiSeda) [reply .openpublishing.publish.config.json:72]: @TomArcherMsft - As in move this whole thing to azure-management-docs-pr or add a block entry for this?

Comment (TomArcherMsft) [reply .openpublishing.publish.config.json:72]: @LiSeda You should always work in the private repo. This PR should be in azure-management-docs-pr.

**Comment by LiSeda** [comment]
Moving to private repo

**Pull request closed without merging**